lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_lint_localizations.rb

```diff
@@ -183,6 +184,14 @@ def self.available_options
             default_value: true,
             type: Boolean
           ),
+          FastlaneCore::ConfigItem.new(
+            key: :fail_on_extra_strings,
+            env_name: 'FL_IOS_LINT_TRANSLATIONS_FAIL_ON_EXTRA_STRINGS',
+            description: 'Should we report violations when finding strings in translations that are not present in the base language',
```

Comment:
How do you feel about being extra verbose here? When reading "extra strings" it wasn't clear to me what the parameter meant.

```suggestion
            key: :fail_on_localized_strings_not_in_base_language,
            env_name: 'FL_IOS_LINT_TRANSLATIONS_FAIL_ON_LOCALIZED_STRINGS_NOT_IN_BASE_LANGUAGE',
            description: 'Should we report violations when finding strings in translations that are not present in the base language',
```

Reply:
Side note. That looked like a very long env var name, but it works, at least on my machine

<img width="1259" alt="image" src="https://github.com/user-attachments/assets/2bff1af0-fb22-408e-a99b-23b6db593421" />

Reply:
> If you can find a shorter name for the parameter that does an equal at explaining its function, I'd be happy with that, too.

Bounced ideas [with ChatGPT](https://chatgpt.com/share/67b3f74e-5f38-8011-9e54-56d2a0242706) and got to a more succinct but clear-enough `fail_on_unmatched_localizations`. What do you think?

Reply:
> succinct but clear-enough fail_on_unmatched_localizations. What do you think?

I like it! Yeah, "missing strings" could potentially be about so many things. I think "unmatched localizations" makes it clearer about what the problem is.

Reply:
> If you can find a shorter name for the parameter that does an equal at explaining its function, I'd be happy with that, too.

Given the context, the term "localized" in "localized string" is implied.
So in fact I think that `fail_on_strings_not_in_base_language` would work.

> That looked like a very long env var name

I'd actually argue that we stop adding `env_name` for each of the arguments in our release-toolkit custom actions. We never use them (who ever set those instead of passing the argument explicitly?).

For some arguments that are common to multiple actions (like `GITHUB_TOKEN`) that makes sense, but for very-specific arguments that only make sense for that specific action, they don't provide anything and are kinda useless to have I'd say 🙃

Reply:
I've renamed the var to `fail_on_strings_not_in_base_language`, thanks for the suggestions @mokagio & @AliSoftware! Updated on e90cd9b4d8234333982493d502dacd5fa3ec09a6.

Reply:
> I'd actually argue that we stop adding env_name for each of the arguments in our release-toolkit custom actions. We never use them (who ever set those instead of passing the argument explicitly?).

Sounds good to me!